mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/IndexedRowMatrix.scala

```diff
@@ -102,6 +102,8 @@ class IndexedRowMatrix(
       k: Int,
       computeU: Boolean = false,
       rCond: Double = 1e-9): SingularValueDecomposition[IndexedRowMatrix, Matrix] = {
+
+    require(k >= 1, "k should be at least one.")
```

Comment:
Could you please make this identical to the message from RowMatrix.computeSVD?  (It gives more info, printing out k and the number of columns.)  It would also be nice to change both to say "numCols" instead of "n"